cylc/uiserver/main.py

```diff
@@ -22,6 +22,8 @@
 import signal
 
 from cylc.flow.network.schema import schema
+from logging.config import fileConfig
```

Comment:
Just noting the from:

https://docs.python.org/3/library/logging.config.html#configuration-file-format

> The fileConfig() API is older than the dictConfig() API and does not provide functionality to cover certain aspects of logging.

Is it worth using `dictConfig` for a better future?

Reply:
My initial implementation was with `dictConfig`. I stopped when I realized that it wouldn't be easy to allow users to extend with their own logging configuration.

I would probably have to do something like

```
if user_provided_config:
    fileConfig(...)
else:
    dictConfig(...)
```

But reading the rest of the docs, looks like `dictConfig` is the way to go. Though, looks like we are actually free to load the `dict` from any resource, even [YAML](https://stackoverflow.com/a/49013493).

I will come up with a different approach with `dictConfig`, but without YAML (as this would imply adding a new dependency to the project). Quite sure we can load a dict from an INI file? Or maybe JSON as that's part of Python stdlib I think.

Reply:
Changed to load from a JSON file. We could hardcode the dictionary as Python code instead (i.e. a Python dictionary). But I thought perhaps this way we can link the file from the docs, saying something like "for logging in Cylc UI Server, you can change its default configuration with a JSON file. See the project default JSON for an example, and the Python logging page for more..."?